internal/driver/glfw/window.go

```diff
@@ -533,10 +531,6 @@ func (w *window) moved(_ *glfw.Window, x, y int) {
 }
 
 func (w *window) resized(_ *glfw.Window, width, height int) {
-	if w.fixedSize {
```

Comment:
Unfortunately Windows fixed size did not work :(

I got it working locally by re-instating this `if w.fixedSize` and setting `width = w.width; height = w.height`.
Then it all worked on windows OS :).

Reply:
I also tested the fix on macOS and it works fine :)